arches/management/commands/packages.py

```diff
@@ -663,6 +663,11 @@ def create_package(self, dest_dir):
                 print("Could not save system settings")
             self.export_package_settings(dest_dir, "true")
 
+    @staticmethod
+    def update_resource_geojson_geometries():
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT * FROM refresh_geojson_geometries();")
```

Comment:
This pg function truncates tables, which you can't do in a transaction, which all our standard Django test cases are wrapped in. De-indenting this allows me to more easily patch it out when we don't need it.

If you've ever seen a wall of text with extra red herring errors after one test failed, it's likely that this wasn't patched out.